infra/scripts/generate-feature-values.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT="$DIR/.."
+
+export PATH=$ROOT/node_modules/.bin:$PATH
+
+cd "$ROOT"
+node -r esbuild-register "$ROOT"/src/feature-env.ts "$@"
+cd -
```

Comment:
You shouldn't have to go back to the previous directory at the end of the script. The context (scope) is restored in the calling script upon completion. However, if you _need_ this, I'd recommend adding an additional subshell. This will :100: always return to the previous directory, even if `node -r esbuild-...` fails.
```suggestion
(
cd "$ROOT"
node -r esbuild-register "$ROOT"/src/feature-env.ts "$@"
)
```